src/experiments.js

```diff
@@ -153,6 +127,32 @@ export function experimentToggles(win) {
   }
 
   Object.assign(toggles_, getExperimentTogglesFromCookie(win));
+
+  if (win.AMP_CONFIG
+      && Array.isArray(win.AMP_CONFIG['allow-url-opt-in'])
+      && win.AMP_CONFIG['allow-url-opt-in'].length > 0) {
+    const allowed = win.AMP_CONFIG['allow-url-opt-in'];
```

Comment:
For me to test what's actually enabled on the page, I need a centralized location of the possible url enabled experiments. Imagine a bunch of `isExperimentOnAllowUrlOverride` calls in the source, how would we know?

Not to mention that if we could figure something out, we would need a dependency on `experiments.js` in the error reporter, which I'd rather avoid.